tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -207,6 +207,7 @@ def test_set_status_after_teardown(self, cluster):
         assert get_status_data["status"] == ResourceServerStatus.terminated
 
     @pytest.mark.level("minimal")
+    @pytest.mark.skip("Test requires terminating the cluster")
```

Comment:
cc @BelSasha with our current test structure we should skip tests that terminate the cluster so following tests can continue running